manager/integration/tests/common.py

```diff
@@ -5375,6 +5375,8 @@ def create_volume_and_write_data(client, volume_name, volume_size=SIZE):
     volume_data = write_volume_random_data(volume)
 
     return volume, volume_data
+
+
```

Comment:
Please put these in the commit `test(auto): system backup/restore`.

Reply:
> Please put these in the commit `test(auto): system backup/restore`.

Hi @c3y1huang :
    I merged it in the commit `test(auto): system backup/restore`.